src/components/StripCard/StripCard.driver.ts

```diff
@@ -0,0 +1,31 @@
+import {
+  BaseUniDriver,
+  baseUniDriverFactory,
+} from 'wix-ui-test-utils/base-driver';
+import { StylableUnidriverUtil, UniDriver } from 'wix-ui-test-utils/unidriver';
+import style from './StripCard.st.css';
+
+export interface stripCardDriver extends BaseUniDriver {
```

Comment:
Super picky, I know, but lets stay with the convention of `PascalCase` for an `interface`'s name.
Meaning `StripCardDriver` instead of `stripCardDriver`

Reply:
it is auto generated so i'm updating the generator too